DisCatSharp.Docs/articles/misc/hosting.md

```diff
@@ -22,6 +22,119 @@ Any modern hardware will work just fine, new or used.
 
 Depending on how complex your bot is, you may even consider purchasing a Raspberry Pi ($35).
 
+# Termux
```

Comment:
The heading doesn't really fit

Reply:
> The headings doesn't really fit

Ah, well no clue.

Reply:
im going to patch that in a bit, thank you still <3